purchases/src/test/java/com/revenuecat/purchases/BasePurchasesTest.kt

```diff
@@ -264,6 +265,23 @@ open class BasePurchasesTest {
         } just Runs
     }
 
+    protected fun mockOfferingsManagerGetOfferings(errorGettingOfferings: PurchasesError? = null): Offerings? {
+        val offerings: Offerings = mockk()
+        every {
+            mockOfferingsManager.getOfferings(appUserId, appInBackground = false, any(), any())
+        } answers {
+            if (errorGettingOfferings == null) {
+                val callback = arg<(Offerings) -> Unit>(3)
```

Comment:
I've become kinda against these ordered parameters... If you changed the parameters, it fails at runtime with a somewhat cryptic message. I would prefer to use slots. Thoughts?

Reply:
ok! yeah, I saw this somewhere else and copied it haha. I will change it to use slots